pkg/autoscaler/multiscaler.go

```diff
@@ -35,6 +35,10 @@ const (
 	// seconds while an http request is taking the full timeout of 5
 	// second.
 	scaleBufferSize = 10
+
+	// SampleSize is the times to scrape metrics per second.
+	// TODO(yanweiguo): tuning the sample size
+	SampleSize = 3
```

Comment:
Over what population?  Is this samples per pod, or samples across all pods?  This comment could use more clarity.

Reply:
Added some more detail in comment.